src/libs/actions/IOU.js

```diff
@@ -1389,23 +1441,25 @@ function setMoneyRequestDescription(comment) {
  * @param {Object[]} participants
  */
 function setMoneyRequestParticipants(participants) {
-    Onyx.merge(ONYXKEYS.IOU, {participants});
+    Onyx.merge(ONYXKEYS.IOU, {
+        participants,
+    });
 }
 
 export {
     deleteMoneyRequest,
-    splitBill,
-    splitBillAndOpenReport,
+    payMoneyRequest,
     requestMoney,
+    resetMoneyRequestInfo,
     sendMoneyElsewhere,
     sendMoneyViaPaypal,
-    payMoneyRequest,
     sendMoneyWithWallet,
-    startMoneyRequest,
-    resetMoneyRequestInfo,
-    setMoneyRequestId,
     setMoneyRequestAmount,
     setMoneyRequestCurrency,
     setMoneyRequestDescription,
+    setMoneyRequestId,
     setMoneyRequestParticipants,
+    splitBill,
+    splitBillAndOpenReport,
+    startMoneyRequest,
```

Comment:
please keep original orders so I can see clearly what are removed and what are newly added